common/memops.c

```diff
@@ -81,6 +82,11 @@
    advice from Fons Adriaensen: make the limits symmetrical
  */
 
+#define SAMPLE_32BIT_MAX   2147483647
+#define SAMPLE_32BIT_MIN   -2147483647
+#define SAMPLE_32BIT_MAX_F  2147483647.0
```

Comment:
These should probably be suffixed with `_D` instead of `_F` because the existing macro names obviously refer to them being float literals.

Reply:
good point!